test/Abp.ZeroCore.Tests/Zero/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -302,6 +302,40 @@ public void Should_Not_Write_History_If_Property_Has_No_Audited_Attribute()
             _entityHistoryStore.DidNotReceive().SaveAsync(Arg.Any<EntityChangeSet>());
         }
 
+        [Fact]
+        public void Should_Not_Write_History_If_Invalid_Entity_Has_Property_With_Audited_Attribute()
+        {
+            //Arrange
+            Post post1 = null;
+
+            /* Category does not inherit from Entity<> */
```

Comment:
```suggestion
            /* Category does not inherit from Entity<> and is not an owned entity */
```

since owned entities that do not inherit from `Entity<>` are still tracked.

Reply:
👍